### ISSUE TITLE
Configure Subplots button implementation in NavigationToolbar missed

### ISSUE THREAD

**Issue by andnovar** [kivy-garden/garden.matplotlib#15]

**Comment by randytbushman** [comment]
I know this is from 5 years ago, but I am currently working on an implementation that the authors will hopefully allow me to merge. It is nearly complete, I am just working on some event handling issues.

**Comment by matham** [comment]
Yeah, just make sure the PR is easy to review (not s giant  PR). And with demo examples so it's easy to test.
